project/Dependencies.scala

```diff
@@ -22,6 +22,8 @@ object Dependencies {
     val akkaHttpBinary = "10.2"
 
     val grpc = "1.37.0" // checked synced by GrpcVersionSyncCheckPlugin
+    // Even referenced explicitly in the sbt-plugin's sbt-tests
+    val googleCommonProtobuf = "3.15.8"
```

Comment:
Or should this just be `protobuf`?

Reply:
It looks like this artifact contains:
```
google/protobuf/descriptor.proto
google/protobuf/struct.proto
google/protobuf/any.proto
google/protobuf/duration.proto
google/protobuf/field_mask.proto
google/protobuf/wrappers.proto
google/protobuf/timestamp.proto
google/protobuf/compiler
google/protobuf/compiler/plugin.proto
google/protobuf/empty.proto
google/protobuf/api.proto
google/protobuf/type.proto
google/protobuf/source_context.proto
```

... so `googleProtobuf` perhaps?